pkg/kf/commands/service-bindings/vcap_services.go

```diff
@@ -18,14 +18,20 @@ import (
 	"encoding/json"
 	"fmt"
 
+	"github.com/google/kf/pkg/kf/cfutil"
 	"github.com/google/kf/pkg/kf/commands/config"
 	"github.com/google/kf/pkg/kf/commands/utils"
 	servicebindings "github.com/google/kf/pkg/kf/service-bindings"
 	"github.com/spf13/cobra"
 )
 
 // NewVcapServicesCommand allows users to bind apps to service instances.
-func NewVcapServicesCommand(p *config.KfParams, client servicebindings.ClientInterface) *cobra.Command {
+func NewVcapServicesCommand(
```

Comment:
I think this command can be removed or changed to read the VCAP_SERVICES from the application's secret instead.

Reply:
It now reads the secret directly